lxd/scriptlet/starlark.go

```diff
@@ -14,25 +15,41 @@ func StarlarkMarshal(input any) (starlark.Value, error) {
 	var err error
 	var sv starlark.Value
 
-	v := reflect.ValueOf(input)
+	if input == nil {
+		return starlark.None, nil
+	}
+
+	if value, ok := input.(starlark.Value); ok {
```

Comment:
Please can you change this to:

```go
value, ok := input.(starlark.Value)
if ok {
}
```

As we don't like to use that sort of inline variable definition.

Reply:
I'll change this and elsewhere in the PR, but as `value` is never used after this `if` statement, this change seems to pollute the function scope?

Reply:
True. You can use a go block {} for that to limit the scope.

Reply:
Okay, I'll add these in here.